docs/getting-started.md

```diff
@@ -1,21 +1,194 @@
+
+*You must use <kbd>ctrl</kbd><kbd>f</kbd> to autocomplete the name and Zulip address of the person you want to PM. If you just type in their full name Zulip Terminal will throw an error.*
+
+<img src="getting-started-imgs/send-pm-autocomplete1.png" width="85%">
+<img src="getting-started-imgs/send-pm-autocomplete2.png" width="85%">
+
+### Create a New Topic
+
+To create a new topic in a stream, type <kbd>c</kbd> and a message editor will pop open at the bottom of the middle column. Make sure your cursor is in the middle column, otherwise nothing will happen when you type <kbd>c</kbd>.
+
+By default this will send a new message to the stream of whichever message you have selected. Let's send a test message to [# test here](https://chat.zulip.org/#narrow/stream/7-test-here). If you're not already in the [# test here](https://chat.zulip.org/#narrow/stream/7-test-here) stream, replace the stream name with 'test here'. Use <kbd>tab</kbd> to jump to the topic section and type in a topic. Type <kbd>tab</kbd> again to jump to the message section and type in a message. Type <kbd>ctrl</kbd><kbd>d</kbd> to send. If you're following along, feel free to actually send the message, [# test here](https://chat.zulip.org/#narrow/stream/7-test-here) is for things like this tutorial.
+
+Type <kbd>esc</kbd> to get out of the message editor.
+
+<img src="getting-started-imgs/new-topic.png" width="85%">
+
+## Edit a Message
+
+Let's try editing the new topic we added to the [# test here](https://chat.zulip.org/#narrow/stream/7-test-here) stream. By default, Zulip allows you to edit the content of your messages within 10 minutes of when you send them, but some servers allow you to edit messages for longer. You can edit message content for up to one hour after sending on the Zulip Community server.
+
+To edit a message, first make sure your cursor is resting on the message you want to edit then type <kbd>e</kbd>. You can edit the message's content or topic, but not the stream. By default your cursor will be in the content portion of the message, press <kbd>tab</kbd> to move it to the stream or topic portion if you'd like to edit those as well. Change whatever you'd like to change, then press <kbd>ctrl</kbd><kbd>d</kbd> to submit the updated message. 
+
+To delete a message, delete all of the message's text.
+
+*Note: Previous versions of the message will still be visible under the message's [edit history](https://zulip.com/help/view-a-messages-edit-history#:~:text=Hover%20over%20a%20message%20to,Click%20View%20edit%20history.).*
+
+<img src="getting-started-imgs/edit-message.png" width="85%">
+
+## Close Zulip Terminal
+
+To exit out of the Zulip Terminal app, type <kbd>ctrl</kbd><kbd>c</kbd>.
+
+## What's Next?
+
+That's all for today! Now that you're acquainted with Zulip Terminal, how about stopping by [our stream](https://chat.zulip.org/#narrow/stream/206-zulip-terminal) and saying hi? We'd love to hear how things went!
```

Comment:
My only thought here is that after all the tutorial, it would be good to point out that if the user has a community server account they can subscribe to #**zulip-terminal** in the webapp and use zulip-terminal to chat in that stream :)

I'm not sure if there's a direct link to go to a subscription management; we don't have subscribing to streams in ZT yet.